example/spec/s3Access/s3AccessSpec.js

```diff
@@ -71,39 +84,63 @@ async function canListObjects(region, credentials) {
 
 const REASON = 'https://bugs.earthdata.nasa.gov/browse/NGAP-4149';
 
+let server;
+
+process.env.AccessTokensTable = `${config.stackName}-AccessTokensTable`;
+const accessTokensModel = new AccessToken();
+
 describe('When accessing an S3 bucket directly', () => {
-  beforeAll(async () => {
+  let accessToken;
+
+  beforeAll(async (done) => {
     await s3().putObject({ Bucket: protectedBucket, Key: testFileKey, Body: 'test' }).promise();
+    setDistributionApiEnvVars();
+    // Use done() callback to signal end of beforeAll() after the
+    // distribution API has started up.
+    server = await serveDistributionApi(config.stackName, done);
   });
 
-  afterAll(async () => {
-    await s3().deleteObject({ Bucket: protectedBucket, Key: testFileKey }).promise();
+  afterAll(async (done) => {
+    try {
+      await s3().deleteObject({ Bucket: protectedBucket, Key: testFileKey }).promise();
```

Comment:
Super minor: use `Promise.all` to parallelize

Reply:
0cda0ce84